### ISSUE TITLE
Match sample length can avoid reading wav files

### ISSUE THREAD

**Issue by haerdogan** [justinsalamon/scaper#89]
It may be possible to avoid reading wav files by just calling soundfile.info on the files.

If the durations match, there would be no need to re-save the file, so it would save running time to not read it and not having to rewrite it when the durations match.

I made an attempt to do this at #86.

**Comment by justinsalamon** [comment]
Yes, we can definitely look into this. At the same time, we're looking a more significant overhaul of the audio processing component in scaper to replace sox entirely, which could impact or already cover this issue.

Lets revisit this issue when the time comes, keeping open.

**Comment by justinsalamon** [comment]
@pseeth will the current PR we're working on (#105) address this issue? I think so?

**Comment by pseeth** [comment]
Yeah it will. Using SoundFile should get rid of off by one errors that this function is a kludge for.

**Comment by justinsalamon** [comment]
OK cool, shall we, as part of #105, get rid of this function altogether if we now can?
